tests/integration/check-custom-repo/test_custom-repos.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
+
+from envparse import env
+
+
+@pytest.mark.good_tests
+def test_good_convertion(shell):
+    convertion = shell(
+        command=[
+            (
+                "convert2rhel -y "
+                "--serverurl {} --username {} "
+                "--password {} --pool {} "
+                "--enablerepo=rhel-server-rpms "
+                "--debug"
+            )
+            .format(
+                env.str("RHSM_SERVER_URL"),
+                env.str("RHSM_USERNAME"),
+                env.str("RHSM_PASSWORD"),
+                env.str("RHSM_POOL"),
+            )
+        ]
+    )
+    assert convertion.returncode == 0
+
+
+@pytest.mark.bad_tests
+def test_with_fake_repo_convertion(shell):
+    convertion = shell(
+        command=[
+            (
+                "convert2rhel -y "
+                "--serverurl {} --username {} "
+                "--password {} --pool {} "
+                "--enablerepo=fake-rhel-server-rpms "
+                "--debug"
+            ).format(
+                env.str("RHSM_SERVER_URL"),
+                env.str("RHSM_USERNAME"),
+                env.str("RHSM_PASSWORD"),
+                env.str("RHSM_POOL"),
+            )
+        ]
+    )
+    assert convertion.returncode == 1
```

Comment:
Let's also capture the log message, so we'll be sure that the conversion fails exactly due to fake-rhel-server-rpms